jsontime/integer.go

```diff
@@ -40,6 +47,32 @@ func (um *UnixMilli) UnmarshalJSON(data []byte) error {
 	return parseTime(data, time.UnixMilli, &um.Time)
 }
 
+func (um UnixMilli) Value() (driver.Value, error) {
+	return time.Time(um.Time).UnixMilli(), nil
+}
+
+func (um *UnixMilli) Scan(src any) error {
+	switch v := src.(type) {
+	case int:
+		um.Time = time.UnixMilli(int64(v))
+	case int8:
+		um.Time = time.UnixMilli(int64(v))
+	case int16:
+		um.Time = time.UnixMilli(int64(v))
+	case int32:
+		um.Time = time.UnixMilli(int64(v))
+	case int64:
+		um.Time = time.UnixMilli(v)
+	default:
+		return fmt.Errorf("%w: %T is not an integer", errors.ErrUnsupported, src)
```

Comment:
This switch should probably be a helper function similar to `parseTime` instead of being duplicated for each precision type